pyperf/_collect_metadata.py

```diff
@@ -92,7 +92,8 @@ def collect_python_metadata(metadata):
         metadata['timer'] = ('%s, resolution: %s'
                              % (info.implementation,
                                 format_timedelta(info.resolution)))
-    elif pyperf.perf_counter == time.clock:
+    elif (hasattr(time, 'clock')
+       and pyperf.perf_counter == time.clock):
```

Comment:
Try to put the elif on a single line. "tox -e pep8" fails.

Reply:
Done.